update_pypi_name_mapping.py

```diff
@@ -0,0 +1,104 @@
+"""
```

Comment:
Would it be possible to put this in cf-scripts (or somewhere else that gets cloned)? Pushing to this repo is a pain since it spawns jobs.

Reply:
doable